### PR TITLE
4 pair classification

### PR DESCRIPTION
**Pull request by GabrielSequeira**
- Add GEM/opusparcus dataset for pair classification eval
- Add Multilingual for pair classification

### CODE REVIEW

File: mteb/abstasks/AbsTaskPairClassification.py [hunk 1]
```diff
@@ -19,9 +21,20 @@ def evaluate(self, model, split="test", **kwargs):
...
+                scores[lang] = self._evaluate_monolingual(model, self.dataset[lang], split, **kwargs)
+        else:
+            logger.info(f"\nTask: {self.description['name']}, split: {split}. Running...")
+            scores = self._evaluate_monolingual(model, self.dataset, split, **kwargs)
+
+        return scores
 
-        logging.getLogger("sentence_transformers.evaluation.PairClassificationEvaluator").setLevel(logging.WARN)
+    def _evaluate_monolingual(self, model, dataset, split, **kwargs):
+        data_split = dataset[split][0]
```
Comment (imenelydiaker) [inline mteb/abstasks/AbsTaskPairClassification.py:36]: Pair Classification is supposed to be multilingual ? I have some trouble understanding the code.. 😢

File: mteb/tasks/PairClassification/OpusparcusPC.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    def __init__(self, langs=None, **kwargs):
+        super().__init__(**kwargs)
+        if type(langs) is list:
+            langs = [lang for lang in langs if lang in self.description["eval_langs"]]
+        if langs is not None and len(langs) > 0:
+            self.langs = langs  # TODO: case where user provides langs not in the dataset
+        else:
+            self.langs = self.description["eval_langs"]
+        self.is_multilingual = True
```
Comment (schmarion) [inline mteb/tasks/PairClassification/OpusparcusPC.py:31]: Is it normal to set mutlilingual always to True ?

File: mteb/abstasks/AbsTaskPairClassification.py (hunk 1 above)
Comment (GabrielSequeira) [reply mteb/abstasks/AbsTaskPairClassification.py:36]: No, it's just the possibility to include datasets who are multilingual. Not cross lingual.

Comment (GabrielSequeira) [reply mteb/abstasks/AbsTaskPairClassification.py:36]: And here i must change the way of loading this dataset with the "quality"
`            self.dataset[lang] = datasets.load_dataset(
                self.description["hf_hub_name"],
                lang=lang,
                quality=100,
                revision=self.description.get("revision", None),
            )`

**Review by imenelydiaker — APPROVED** [review]

**Review by schmarion — APPROVED** [review]

**Pull request merged**